app/controllers/folders/collaborations_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Folders::CollaborationsController < ApplicationController
+  def new
+    @current_folder = Folder.find(params[:folder_id])
+    @collaboration  = @current_folder.collaborations.new
+  end
+
+  def create
+    @current_folder = Folder.find(params[:folder_id])
```

Comment:
With how often we're calling this method now, it should definitely be a helper method that we call with a before_action 👍